packages/eslint-plugin-orbit-components/src/rules/defaultTheme.ts

```diff
@@ -0,0 +1,126 @@
+import * as t from "@babel/types";
+import type { Rule } from "eslint";
+
+export const ERRORS = {
+  variableDeclaration: "Do not use defaultTheme as value, use useTheme() hook or ThemeConsumer",
+  styled:
+    "Do not use defaultTheme directly inside styled components, either create defautlProps or consume the theme from with arrow function expression.",
+  import: "Do not name defaultTheme import as theme, please rename it to defaultTheme/themeDefault",
+  desctructured: "Do not destructure defaultTheme, use useTheme() hook or ThemeConsumer",
```

Comment:
Also update in the test file.
```suggestion
  destructured: "Do not destructure defaultTheme, use useTheme() hook or ThemeConsumer",
```